.gitignore

```diff
@@ -1,5 +1,5 @@
+lib-test
 dist
-lib
```

Comment:
Yes. We could do that too once we are publishing it on npm (in which case we could opt to just publish /lib). Though for now since its mainly a github based repo we would need it in the repo.

Would you want to just move to publishing to NPM and have people try it out from there? We'd need to work out a decent git tagging and npm publish workflow going for that.